assets/locales/en/translation.json

```diff
@@ -61,6 +61,9 @@
       "TableDropdown": {
         "New SQL query": "New SQL query",
         "Remove": "Remove",
+        "Are you sure to delete table `{{name}}`?": "Are you sure to delete table `{{name}}`?",
```

Comment:
I'd make the texts shorter like so:
"Remove table {{name}}?"
"Remove"
"Cancel"

I think yes/no modals are to be avoided. We're using "remove" in the GUI, and "delete" in my ear sounds more like destruction of the data, not just removing from some form of "registry" which our app kinda is.